### DISCUSSION TITLE
njuthesis-setup.def 配置文件的一个小问题

### DISCUSSION THREAD

**Discussion by Soulike** [nju-lug/NJUThesis#211] [General]
![image](https://github.com/nju-lug/NJUThesis/assets/23612560/7539a22c-bb52-4a1d-bfc6-873df1237ad7)
模版里英文学位名称配置项 `degree*` 后面少了逗号，解注释就会报错

**Comment by atxy-blip** [comment]
<https://github.com/nju-lug/NJUThesis/commit/5fcd8ab82664a7363517739706783d69306cebdf> 改了。
